### PR TITLE
Change from JDK9 to JDK11

### LINKED ISSUE #19: JDK 10 and 11 support
JDK 9 is gone.

### PR DESCRIPTION
**Pull request by mdedetrich**
Currently this project appears to be hardcoded to use JDK 9 as the alternative JDK for multi release, this PR changes this to JDK 11. The main intent behind this change is that JDK 9 is a non LTS release and because of this its actually cumbersome to install on various platforms. This also opens up the project to use github actions for CI.

Due to changes between JDK 9 and JDK 11 there was some additional work that needed to be done to get `sbt scripted` to pass, notable changes are

* I had to add a `SecurityPolicy` `policy` file as well enable the `SecurityManager`. While the `SecurityManager` problems were solved in later versions of sbt, updating this projects sbt version causes the tests to fail and so its best to solve this problem separately. This change will be reverted once the sbt version is updated to a later one.
* The flags for specifying `source` and `target` had to be changed slightly since the format was updated since JDK 11

This PR also adds `-SNAPSHOT` to the version we can properly iterate changes and test them.

One argument that may pop up is that we are essentially dropping support for JDK9, I think a cleaner/more proper way to solve this would be https://github.com/sbt/sbt-multi-release-jar/issues/24 .

Resolves: https://github.com/sbt/sbt-multi-release-jar/issues/19

### CODE REVIEW

File: .travis.yml
Comment (eed3si9n) [inline .travis.yml:1]: We no longer test using Travis CI, so maybe we should get CI going first?

**Comment by ktoso** [comment]
Don't wait for me on reviews, I can't contribute to technical discussion without extra permissions which take time, so go ahead with merging when you're happy.

File: .travis.yml
Comment (mdedetrich) [reply .travis.yml:1]: Actually thats the reason for doing this PR first, I wanted to use stable JDK versions for the github actions CI and currently the entire project is hardcoded to use JDK 9.

Idea is after this PR is merged I will immediately set up github actions CI

**Review by eed3si9n — APPROVED** [review]

**Pull request merged**
